tests/js/browser/all-browser-tests.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+var sequence = require("when/sequence");
+var tests = [
+    "./browser-binder-tests.js",
+    "./browser-initBlock-tests.js",
+    "./browser-inline-tests.js",
+    "./browser-rendering-tests.js",
+    "./browser-templateAware-tests.js",
+    "./browser-templateFormControl-tests.js",
+    "./browser-templateMessage-tests.js"
+];
+var promises = [];
```

Comment:
clearer as var promises = fluid.transform(tests, function (test) {
    return require(test);
});

If you have become an ES5 purist you can also use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map